### ISSUE TITLE
Upgrade to Swift 4.0+

### ISSUE THREAD

**Issue by eggli** [pushy-me/pushy-react-native#14]
In the Xcode 10.2 released today, Swift 3.0 support is now dropped by Apple, we will unable to build pushy-react-native installed React Native iOS project anymore, please upgrade to Swift 4.0+.

**Comment by rizwankce** [comment]
@eggli This [commit](https://github.com/pushy-me/pushy-react-native/commit/2893f41aea92964b5d065901cae9d610ae9bcefc) seems to support for Swift 4.0 from Pushy SDK. But I think the Xcode project is not yet updated

**Comment by pushy-me** [comment]
Thanks @eggli, merged and published in `1.0.8`.

To update, please run the following command in your root project directory:
```
npm install pushy-react-native@1.0.8 --save
```

Run the following command to relink the iOS native module to your project:
```
react-native unlink pushy-react-native
react-native link pushy-react-native
```

Then, recompile and let us know whether the issue is resolved.

**Comment by eggli** [comment]
Thanks for your quick response, this issue has been resolved.
